xcdat/spatial_avg.py

```diff
@@ -441,40 +500,39 @@ def _align_longitude_to_360_axis(self, domain_bounds: xr.DataArray):
         This method returns ``domain_bounds`` that are intended for calculating
         spatial weights only.
         """
-
-        # copy bounds to work on
-        d_bounds = domain_bounds.copy()
```

Comment:
Removed this line since copying is being done in the bounds dict in get_weights().